packages/shared/package.json

```diff
@@ -2,9 +2,27 @@
   "name": "@clerk/shared",
   "version": "1.0.0",
   "description": "Internal package utils used by the Clerk SDKs",
-  "sideEffects": false,
```

Comment:
Did you notice any side-effects (pun intended) in regards to code pruning when removing this?

Reply:
Still all worked fine during bundling :)